prov/verbs/src/verbs_info.c

```diff
@@ -361,7 +361,8 @@ int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 		return -FI_ENODATA;
 	}
 
-	if (attr->total_buffered_recv > info->rx_attr->total_buffered_recv) {
+	if (!attr->total_buffered_recv &&
+		(attr->total_buffered_recv > info->rx_attr->total_buffered_recv)) {
```

Comment:
@a-ilango thanks for clarification! I added FI_RM_ENABLED for EP_RDM and corresponding check to total_buffered_recv comparison